syntax = "proto3";

package hello;

// The greeting service definition
service HelloService {
  // Sends a greeting and gets a response
  rpc GetHelloMessage (HelloRequest) returns (HelloReply) {}
}

// The request message containing the client's name
message HelloRequest {
  string message = 1;
}

// The response message containing the server's response
message HelloReply {
  string message = 1;
}
